tide_media.module

```diff
@@ -337,3 +346,42 @@ function tide_media_entity_embed_alter(array &$build, EntityInterface $entity, a
     }
   }
 }
+
+/**
+ * Custom validation.
+ */
+function tide_media_form_media_embedded_video_validate($form, FormStateInterface $form_state) {
+  if ($form_state->hasValue(['inline_entity_form'])) {
```

Comment:
hey @edyuenyw ,  you might want to check if the current page is `embedded_video` page, otherwise, it will affect other iframe